### ISSUE TITLE
entr does not stop when the terminal is closed 

### ISSUE THREAD

**Issue by tobiasBora** [eradman/entr#57]
Hello!

When I run entr, for example using this command:

```
$ touch /tmp/ab
$ echo /tmp/ab | entr -n -r -s "echo New; sleep 123456789"
```
I can check in another terminal that the command is running:
```
$ ps aux | grep 123456789
me        445  0.0  0.0   6428   996 pts/4    S+   12:51   0:00 entr -n -r -s echo New; sleep 123456789
me        446  0.0  0.0 219012  2560 pts/4    S    12:51   0:00 /nix/store/xadrr3l5jvkkm3g3lb2g81j5wz51zqdv-bash-interactive-4.4-p23/bin/bash -c echo New; sleep 123456789 /tmp/ab
me        447  0.0  0.0 218776   320 pts/4    S    12:51   0:00 sleep 123456789
```
If I type `Ctrl-C`, the processes spawn by `entr` are killed, which is great. I can see it by running again:
```
$ ps aux | grep 123456789
```

However, if I close my terminal (`konsole` in my case), the process is not killed:
```
$ ps aux | grep 123456789
me        446  0.0  0.0 219012  2560 ?        S    12:51   0:00 /nix/store/xadrr3l5jvkkm3g3lb2g81j5wz51zqdv-bash-interactive-4.4-p23/bin/bash -c echo New; sleep 123456789 /tmp/ab
me        447  0.0  0.0 218776   320 ?        S    12:51   0:00 sleep 123456789
```

You can see that it is not attached to any pseudo terminal, and if you check which are the ancestors of `entr`, you will see that it's systemd process with pid `1`.

Is this a feature or a bug? If it's a feature, is it possible to disable it? Indeed, I use `entr` to update a server, and sometimes the terminal get closed, usually when I open the terminal from emacs and restart emacs to update my configuration. And then, I cannot restart my server (port already in use) without killing all the spawn processes, which is quite annoying.

**Comment by eradman** [comment]
Thanks for the the easy cut-and-paste example!

`entr` did handle `SIGHUP`. I implemented this in commits 1ee465578985bcca0821ba07233839cedbaf4d78, 15cf75cf9602acf151ae047533c670ec4035e2d4

**Comment by eradman** [comment]
This change was finally tagged in the 4.7 release today
